source/_integrations/glinet.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title: GL-inet
+description: Instructions on how to integrate a GL-inet router into Home Assistant.
+ha_category:
+  - Hub
+  - Presence Detection
+ha_release: 2021.8
+ha_config_flow: true
+ha_iot_class: Local Polling
+ha_codeowners:
+  - '@HarvsG'
+ha_domain: glinet
+ha_platforms:
+  - device_tracker
+---
+
+GL-inet are a Hong-Kong based company who produce highly customizable home routers. They are based on the very common OpenWRT firmware but provide a much more user-friendly user interface than OpenWRT's LUCI.
```

Comment:
Can we link them in the text?

```suggestion
GL-inet is a Hong-Kong based company that produces highly customizable home routers. They are based on the very common OpenWRT firmware but provide a much more user-friendly user interface than OpenWRT's LUCI.
```

Reply:
Done in https://github.com/home-assistant/home-assistant.io/pull/18537/commits/ac3fd913b97a91904c1c1791810b24e90c6dac43 and https://github.com/home-assistant/home-assistant.io/pull/18537/commits/051ee9f244f78c3fa9538fb3aa15f585ca0c352c